### PR TITLE
Revert changes to StringCodec and fix script compilation.

### PR DESCRIPTION
**Pull request by johnou**
Fixes regressions introduced by https://github.com/redisson/redisson/commit/4a25fab9df7ee9ef84416609c445ff9b12b3238a

### CODE REVIEW

File: redisson/src/main/java/org/redisson/client/codec/StringCodec.java [hunk 1]
```diff
@@ -41,16 +40,9 @@
...
-                ByteBuf buf = ByteBufAllocator.DEFAULT.buffer(ByteBufUtil.utf8MaxBytes(payload));
-                ByteBufUtil.writeUtf8(buf, payload);
-                return buf;
-            } else {
-                ByteBuf out = ByteBufAllocator.DEFAULT.buffer();
-                out.writeCharSequence(in.toString(), charset);
-                return out;
-            }
+            ByteBuf out = ByteBufAllocator.DEFAULT.buffer();
+            out.writeCharSequence(in.toString(), charset);
```
Comment (johnou) [inline redisson/src/main/java/org/redisson/client/codec/StringCodec.java]: I'll need to investigate this further, it caused some tests to fail.

**Comment by johnou** [comment]
@mrniko one test left to fix (org.redisson.RedissonMapCacheTest#testGetAll).

**Comment by mrniko** [comment]
Sorry, I didn't get. Why RedissonMapCache has been also changed?

**Comment by johnou** [comment]
@mrniko I reverted a lot of the spacing changes because one of them caused a script error.

**Comment by johnou** [comment]
@mrniko I reverted a lot of the spacing changes because one of them caused a script error. I was on the wrong branch when running tests and only realised after you merged.

**Comment by johnou** [comment]
@mrniko there is also a off by one error in MapGetAllDecoder which is now fixed.

**Pull request merged**